CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.3.4 (2024-02-09)
```

Comment:
😭 changelog generation is a part of "yarn", but not always a part of "build".
So it constantly getting outdated :(